pkg/sqlite/migrationsPostgres/1_initial.up.sql

```diff
@@ -0,0 +1,475 @@
+  scene_id integer not null,
+  o_date timestamp not null,
+  foreign key(scene_id) references scenes(id) on delete CASCADE
+);
+CREATE TABLE performer_stash_ids (
+  performer_id integer,
+  endpoint varchar(255),
+  stash_id varchar(36),
+  foreign key(performer_id) references performers(id) on delete CASCADE
+);
+CREATE TABLE studio_stash_ids (
+  studio_id integer,
+  endpoint varchar(255),
+  stash_id varchar(36),
+  foreign key(studio_id) references studios(id) on delete CASCADE
+);
+CREATE TABLE tags_relations (
+  parent_id integer,
+  child_id integer,
+  primary key (parent_id, child_id),
+  foreign key (parent_id) references tags(id) on delete cascade,
+  foreign key (child_id) references tags(id) on delete cascade
+);
+CREATE TABLE files_fingerprints (
+  file_id integer NOT NULL,
+  type varchar(255) NOT NULL,
+  fingerprint text NOT NULL,
+  foreign key(file_id) references files(id) on delete CASCADE,
+  PRIMARY KEY (file_id, type, fingerprint)
+);
+CREATE TABLE video_files (
+  file_id integer NOT NULL primary key,
+  duration float NOT NULL,
+	video_codec varchar(255) NOT NULL,
+	format varchar(255) NOT NULL,
+	audio_codec varchar(255) NOT NULL,
+	width smallint NOT NULL,
+	height smallint NOT NULL,
+	frame_rate float NOT NULL,
+	bit_rate integer NOT NULL,
+  interactive boolean not null default FALSE,
+  interactive_speed int,
+  foreign key(file_id) references files(id) on delete CASCADE
+);
+CREATE TABLE video_captions (
+  file_id integer NOT NULL,
+  language_code varchar(255) NOT NULL,
+  filename varchar(255) NOT NULL,
+  caption_type varchar(255) NOT NULL,
+  primary key (file_id, language_code, caption_type),
+  foreign key(file_id) references video_files(file_id) on delete CASCADE
+);
+CREATE TABLE image_files (
+  file_id integer NOT NULL primary key,
+  format varchar(255) NOT NULL,
+  width smallint NOT NULL,
+	height smallint NOT NULL,
+  foreign key(file_id) references files(id) on delete CASCADE
+);
+CREATE TABLE images_files (
+    image_id integer NOT NULL,
+    file_id integer NOT NULL,
+    "primary" boolean NOT NULL,
+    foreign key(image_id) references images(id) on delete CASCADE,
+    foreign key(file_id) references files(id) on delete CASCADE,
+    PRIMARY KEY(image_id, file_id)
+);
+CREATE TABLE galleries_files (
+    gallery_id integer NOT NULL,
+    file_id integer NOT NULL,
+    "primary" boolean NOT NULL,
+    foreign key(gallery_id) references galleries(id) on delete CASCADE,
+    foreign key(file_id) references files(id) on delete CASCADE,
+    PRIMARY KEY(gallery_id, file_id)
+);
+CREATE TABLE scenes_files (
+    scene_id integer NOT NULL,
+    file_id integer NOT NULL,
+    "primary" boolean NOT NULL,
+    foreign key(scene_id) references scenes(id) on delete CASCADE,
+    foreign key(file_id) references files(id) on delete CASCADE,
+    PRIMARY KEY(scene_id, file_id)
+);
+CREATE TABLE IF NOT EXISTS performers_scenes (
+  performer_id integer,
+  scene_id integer,
```

Comment:
Should these be non-nullable?

Reply:
Ehh, they are written like that in the original schema, and they are foreign keys that reference other tables.
So i think its fine